tutorials/scripting/gdscript/gdscript_basics.rst

```diff
@@ -13,6 +13,8 @@ create content. It uses a syntax similar to
 to be optimized for and tightly integrated with Godot Engine, allowing great
 flexibility for content creation and integration.
 
+GDScript is entirely independent from Python and is not based on it.
```

Comment:
I think this sounds like a denial now 🙂 Maybe the paragraph itself should be rephrased instead, like you did for the Godot design philosophy? If we put "indent-based" to the first place, and move "similar to languages like Python" in the parentheses, and also remove "many keywords are similar", it will be less conflicting.